docs/request.md

```diff
@@ -202,23 +203,23 @@ Request URLs may contain interpolations:
 
 ```javascript
 m.request({
-	method: "GET",
-	url: "/api/v1/users/:id",
-	data: {id: 123},
+  method: "GET",
+  url: "/api/v1/users/:id",
+  params: {id: 123},
 }).then(function(user) {
-	console.log(user.id) // logs 123
+  console.log(user.id) // logs 123
 })
 ```
 
 In the code above, `:id` is populated with the data from the `{id: 123}` object, and the request becomes `GET /api/v1/users/123`.
 
-Interpolations are ignored if no matching data exists in the `data` property.
+Interpolations are ignored if no matching data exists in the `params` property.
 
 ```javascript
 m.request({
-	method: "GET",
-	url: "/api/v1/users/foo:bar",
-	data: {id: 123},
+  method: "GET",
+  url: "/api/v1/users/foo:bar",
+  params: {id: 123},
 })
 ```
 
```

Comment:
```suggestion

In the code above, the request becomes `GET /api/v1/users/foo:bar?id=123`.
```

Reply:
This particular suggestion is optional.

(Including this purely for the sake of review.)

Reply:
I've included it - I believe it is important, so users will know that if their `params` contains a lot of properties, they will all be added to the URL.